src/service/tempMailApi.js

```diff
@@ -14,15 +14,16 @@ class TemporalMailService {
       const responseDomain = await this._getDomain();
       const domain = responseDomain['hydra:member'][0].domain;
 
+
       const response = await this.httpClient.post(`${this.BASE_API_URL}/accounts`, {
         address: `${username}@${domain}`,
         password,
       });
 
       return response.data;
     } catch (err) {
-      if (err.response.data['hydra:description'].startsWith('address: The username') &&
-      err.response.data['hydra:description'].endsWith('is not valid.')) {
+      if (err.response && err.response.data['hydra:description'].startsWith('address: This value') &&
```

Comment:
err.response && err.response.data .... Why?

Reply:
Because, there are sometimes that the error i get is that cannot get data from undefined because some errors dont have the property "response" so i added that extra condition so it doesnt crash when we get error without responses.